qiskit/backends/ibmq/ibmqbackend.py

```diff
@@ -51,9 +55,27 @@ def run(self, qobj):
         Args:
             qobj (dict): description of job
 
+        Raises:
+            ValueError: if qobj is invalid
+
         Returns:
             IBMQJob: an instance derived from BaseJob
         """
+
+        # verify the QObj is valid before making requests for computing resources
+        sdk = qiskit_path[0]
+        # Schemas path:     qiskit/backends/schemas
+        schemas_path = os.path.join(sdk, 'schemas')
+        schema_file_path = os.path.join(schemas_path, 'qobj_schema.json')
+
+        with open(schema_file_path, 'r') as schema_file:
+            schema = json.load(schema_file)
```

Comment:
Since this only needs to be done once, loading the schema could be moved to `__init__`.